akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowSpec.scala

```diff
@@ -489,11 +489,10 @@ class FlowSpec extends AkkaSpec(ConfigFactory.parseString("akka.actor.debug.rece
         downstream2.expectNext("a3")
         downstream2.expectComplete()
 
-        // FIXME when adding a sleep before the following link this will fail with IllegalStateExc shut-down
-        // what is the expected shutdown behavior? Is the title of this test wrong?
-        //        val downstream3 = StreamTestKit.SubscriberProbe[Any]()
-        //        publisher.subscribe(downstream3)
-        //        downstream3.expectComplete()
+        val downstream3 = StreamTestKit.SubscriberProbe[Any]()
+        publisher.subscribe(downstream3)
+        downstream3.expectSubscription()
+        downstream3.expectError().getMessage should ===("Cannot subscribe to shut-down Publisher")
```

Comment:
This message is defined in ActorPublisher.NormalShutdownException. Please extract it to a val and compare with that here. It is also written in StreamOfStreamProcessors.scala and SubscriberManagement.scala. Looks like they could use the ActorPublisher.NormalShutdownException instead.